### ISSUE TITLE
Maps Examples Won't Load in HoloLens 1 (Maps ver 0.1.5)

### ISSUE THREAD

**Issue by cobrauf** [microsoft/MapsSDK-Unity#9]
I have deployed the weather map example scene from Maps ver 0.1.3, and map data loads successfully in Hololens.

I tried doing the same thing using Maps ver 0.1.5 and map data does not load in Hololens. I just see the empty grid on top of the purple maps base cube.

I have both apps on the Hololens and 0.1.3 loads but 0.1.5 does not, which rules out any issues with the Hololens.

I have compared project settings on both projects and they mirror exactly, so I cannot figure out what's the issue.

For both:
Unity 2018.3.4
VS 2019, Windows 10 Insider Preview
Hololens 1
Target SDK 18362
Min Platform version 10240

Thanks!

**Comment by kircher1** [comment]
Thanks for the report! I'll take a look and try and repro.

**Comment by kircher1** [comment]
Which Scripting Backend (in Player Settings -> Other Settings) does your project use, IL2CPP or .NET?

**Comment by cobrauf** [comment]
IL2CPP (thanks for investigating so quickly)

**Comment by cobrauf** [comment]
[UnityPlayer-Maps-0.1.3.log](https://github.com/microsoft/MapsSDK-Unity/files/3184702/UnityPlayer-Maps-0.1.3.log)
[UnityPlayer-Maps-0.1.5.log](https://github.com/microsoft/MapsSDK-Unity/files/3184703/UnityPlayer-Maps-0.1.5.log)

Here are the UnityPlayer logs from each app, in case that helps.

**Comment by kircher1** [comment]
Thanks for sharing the logs!

In the 0.1.5 log that you shared, there is a networking error hidden in there: "Cannot resolve destination host". This indicates that the device has no network connection or the app does not have "internet" capability enabled. (The map requires an internet connection to request and download data from the servers.) 

I was able to reproduce this message by disabling the internet connection on the HoloLens.

Double-check the following:
1) The device has an internet connection
2) The app capabilities should include "InternetClient". (This can be found under the Player Settings -> Publishing Settings -> Capabilities.)

If neither of those solves what you are seeing, then try upgrading to the latest version of 2018.3. (Unlikely that is the issue, but I'm using 2018.3.12.)

**Comment by cobrauf** [comment]
Thanks for catching that in the logs, I doubled checked my Unity capability settings and I in fact have InternetClient enabled, but in the Package.appmanifest file in the VS .sln project it was not enabled, so somehow the capabilities selection was not getting carried over from Unity to VS (strangely, spatial perception was carried over).  I enabled InternetClient  in Package.appmanifest > Capabilities and now Maps loads successfully.

Thanks for the help!
